hw/dv/sv/bus_params_pkg/bus_params_pkg.core

```diff
@@ -0,0 +1,19 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:bus_params_pkg"
```

Comment:
Can we make this name contain "OpenTitan"? Since each project that uses the DV code needs to define its own core that supplies `bus_params_pkg`, I reckon OpenTitan shouldn't get the "canonical name" :-)

In particular, when vendoring code back in from Ibex we'll probably pick up Ibex's bus params package too so they'll need to have distinct names.

To make this work when we vendor the DV code *in* to Ibex, we'll patch its dependency to change from "opentitan bus params" to "ibex bus params", in the same way as we're currently patching the top_pkg dependency. (And, of course, this work lets us get rid of the top_pkg dependency in Ibex entirely).

Reply:
Does `lowrisc:opentitan:bus_params_pkg` work?

Reply:
Looks good to me! Sorry for the slow reply: I was on holiday last week. Thanks for doing this.